cake-autorate.sh

```diff
@@ -32,26 +30,23 @@ cleanup_and_killall()
 	log_msg "INFO" ""
 	log_msg "INFO" "Killing all background processes and cleaning up temporary files."
 
-	if ! [[ -z ${maintain_pingers_pid} ]]; then
+	if [[ -n ${maintain_pingers_pid} ]]; then
```

Comment:
BTW what's the reason to switch over to -n rather than ! -z?

Reply:
Really there is no reason, it just looks less confusing style-wise